test/api_integration/apis/saved_objects/bulk_create.ts

```diff
@@ -72,6 +72,7 @@ export default function ({ getService }: FtrProviderContext) {
                 },
                 coreMigrationVersion: '8.8.0',
                 typeMigrationVersion: resp.body.saved_objects[1].typeMigrationVersion,
+                managed: resp.body.saved_objects[1].managed,
```

Comment:
The `bulkCreate` api accepts a `managed` option that is applied to all objects being created. The option overrides whatever's already set on a document, so it won't matter what value we assign to the `BULK_REQUESTS`.

The intended behavior is as follows:
-   If `options.managed` is set, it will override any existing `managed` value in all the documents being created. (needed for `import` - he only intended way to override the `managed` default)
-   If `options.managed` is not provided, then the documents are created with whatever may be assigned to their `managed` property or default to `false`.

A test for that will definitely be good.

Reply:
Another thing I need to point out again is that we're not exposing setting the `managed` option to HTTP requests. The property will only be settable (if that's a word (🤷‍♀️ ) ) using the server-side `import` saved objects API.

Reply:
> we explicitly don't want to expose the option to set managed over HTTP

Sorry, I assumed we wanted that, but thinking about it it makes sense to not support it.

In that case the tests for write API operations (create/bulkCreate) are probably fine.

We'll still want to update the read API operations (get, bulkGet) tests, but that can be done in the next PR, when we can actually inject data (with this managed flag being set) via import/export.

Reply:
Ok, I'll omit the commits where I implemented the workaround, _unless_ @rudolf feels otherwise.